virtual-desktop/nodeServer/src/browserPreferences.ts

```diff
@@ -23,7 +23,7 @@ class BrowserPreferencesDataservice {
     this.context = context;
     const router = express.Router();
     router.use(function noteRequest(req: Request, res: Response, next: any): void {
-      context.logger.info('Saw request, method=' + req.method);
+      context.logger.info('ZWED0002I', req.method); /*context.logger.info('Saw request, method=' + req.method);*/
```

Comment:
This will not work, because you do not have a translation JSON for any lib or nodeserver content.

Reply:
Missed this